var/spack/repos/builtin/packages/coreneuron/package.py

```diff
@@ -155,6 +158,11 @@ def get_cmake_args(self):
              '-DPYTHON_EXECUTABLE=%s' % spec["python"].command.path
              ]
 
+        # Versions after this only used C++, but we might still need C
+        # flags if mod2c is being built as a submodule.
```

Comment:
Shouldn't NMODL always be external?

Reply:
It is, yes. But if NMODL is disabled we build mod2c internally, and mod2c uses C. Old CoreNEURON itself also used C.

Reply:
Sigh… alright. I get my stuff a wee bit confused.